x-pack/legacy/plugins/canvas/canvas_plugin_src/expression_types/embeddable.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ExpressionType } from '../../../../../../src/plugins/data/common/expressions';
```

Comment:
Webpack creates an alias for these that allow you to remove the leading `../`.  For example:

https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/browser/location.ts#L7

Reply:
Did changing this to `src/plugins/data/common/expressions` not work for you?

Reply:
Yes, the alias works here, but there are some other files where it does not appear to work, the `embeddable_types` file that is in the same directory as this file for instance gives module not found errors when using the `src/legacy/...` path instead of the full relative path. So I'm just unsure when and when to not do relative paths.  I've fixed all the paths that don't cause errors when changing them to use the alias.